.github/workflows/test.yaml

```diff
@@ -1,15 +1,15 @@
 name: Test
 
 on:
+  workflow_dispatch:
   push:
-    branches: [ 'main', 'release-*' ]
+    branches: ['main', 'release-v*']
   pull_request:
     branches: ['*']
   merge_group:
     types: [ checks_requested ]
   schedule:
     - cron: "15 1 * * *"
-  workflow_dispatch:
```

Comment:
Why remove? I think it is handy to run the tests anytime.

Reply:
it's hoisted, at line [4](https://github.com/Kuadrant/kuadrant-operator/pull/1197/files#diff-245392b692a50c38ecab4381b118862db514035c10983f3bd4f4b7f1f4be4692R4 ) ... sorry, stupid change